measures/2019/measures-data.json

```diff
@@ -2488,7 +2488,7 @@
     }
   },
   {
-    "title": "Colonoscopy Interval for Patients with a History of Adenomatous Polyps \r- Avoidance of Inappropriate Use",
+    "title": "Colonoscopy Interval for Patients with a History of Adenomatous Polyps\\r\\n \r- Avoidance of Inappropriate Use",
```

Comment:
Looks like we still have some sort of carriage return (unescaped) in here. Is that intentional?